packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/utils.test.js

```diff
@@ -27,29 +27,29 @@ import traceGenerator from '../../../demo/trace-generators';
 describe('TraceTimelineViewer/utils', () => {
   describe('getViewedBounds()', () => {
     it('works for the full range', () => {
-      const args = { min: 1, max: 2, start: 1, end: 2, viewStart: 0, viewEnd: 1 };
-      const { start, end } = getViewedBounds(args);
+      const args = { min: 1, max: 2, viewStart: 0, viewEnd: 1 };
+      const { start, end } = createViewedBoundsFunc(args)(1, 2);
       expect(start).toBe(0);
       expect(end).toBe(1);
     });
 
     it('works for a sub-range with a full view', () => {
-      const args = { min: 1, max: 2, start: 1.25, end: 1.75, viewStart: 0, viewEnd: 1 };
-      const { start, end } = getViewedBounds(args);
+      const args = { min: 1, max: 2, viewStart: 0, viewEnd: 1 };
+      const { start, end } = createViewedBoundsFunc(args)(1.25, 1.75);
       expect(start).toBe(0.25);
       expect(end).toBe(0.75);
     });
 
     it('works for a sub-range that fills the view', () => {
-      const args = { min: 1, max: 2, start: 1.25, end: 1.75, viewStart: 0.25, viewEnd: 0.75 };
-      const { start, end } = getViewedBounds(args);
+      const args = { min: 1, max: 2, viewStart: 0.25, viewEnd: 0.75 };
+      const { start, end } = createViewedBoundsFunc(args)(1.25, 1.75);
       expect(start).toBe(0);
       expect(end).toBe(1);
     });
 
     it('works for a sub-range that within a sub-view', () => {
       const args = { min: 100, max: 200, start: 130, end: 170, viewStart: 0.1, viewEnd: 0.9 };
```

Comment:
you can remove the `start` and `end` args

Reply:
good catch, though I had caught them all.